components/index.js

```diff
@@ -3,6 +3,8 @@ export { default as Dashicon } from './dashicon';
 export { default as FormToggle } from './form-toggle';
 export { default as IconButton } from './icon-button';
 export { default as Panel } from './panel';
+export { default as PanelHeader } from './panel/header';
+export { default as PanelBody } from './panel/body';
```

Comment:
The problem with `import PanelBody from 'components/panel/header'` is that it will cause the code to be duplicated across builds. This may be solvable via some trick I don't know about, see #941.